library/init/meta/simp_tactic.lean

```diff
@@ -372,7 +372,7 @@ meta def to_simp_lemmas : simp_lemmas → list name → tactic simp_lemmas
 | S (n::ns) := do S' ← S^.add_simp n, to_simp_lemmas S' ns
 
 meta def mk_simp_attr (attr_name : name) : command :=
-do t ← to_expr ``(caching_user_attribute simp_lemmas),
+do let t := ```(caching_user_attribute simp_lemmas),
    v ← to_expr ``({name     := %%(quote attr_name),
```

Comment:
I'm also not sure. Do you have a name for it? 
We would also be able to add instances for `nat`, `string`, `list name`, `list nat`, `list string`, etc.
It may look weird to have multiple instances of `list`.
Do you see a general solution for this?